### DISCUSSION TITLE
SHINNY APPS

### DISCUSSION THREAD

**Discussion by Anggitnoviee** [teamalgoritma/community#213] [Q&A] [answered]
Bagaimana cara shinny apps connect to website internal perusahaan??

**Comment by idoali** [comment, marked as answer]
Selamat Pagi bu

Bisa bu. Ada beberapa pilihan yang Ibu bisa gunakan yang sekarang bisa kami singgung.

1. Menggunakan Docker. Dengan menggunakan Docker, Ibu bisa membuat container yang nantinya bisa dijalankan dimana saja, termasuk ke website internal kantor. Informasi lebih lanjut bisa diliat disini bu : https://www.r-bloggers.com/2021/05/dockerizing-shiny-applications/#:~:text=All%20the%20general%20advantages%20of,the%20same%20in%20the%20future.

2. Menggunakan ShinyServer. Ini adalah program untuk shiny yang dapat digunakan menjadi web server. Informasi lebih lanjut : https://shiny.rstudio.com/tutorial/written-tutorial/lesson7/ 

Berikut jawaban dari kami bu. Semoga bisa membantu

**Reply by Anggitnoviee** [reply]
Terimakasi
